<!-- launches one node for the specified tag under namespace /uwb/twr/TAG_ID
the data from the tag is therefore published on topics /uwb/twr/TAG_ID/distances-->

<launch>
    <arg name="usb_port" default="/dev/ttyACM0"/>
    <arg name="read_freq" default="100"/>
    <arg name="world_frame" default="world"/>
    <arg name="publish_tfs" default="true"/>

    <group>
        <node name="$(anon uwb_tag)" pkg="dwm1001c_ros" type="uwb_tag.py" args="$(arg usb_port) $(arg read_freq)" output="screen" required='true'/>
        <node name="$(anon uwb_pose_publisher)" pkg="dwm1001c_ros" type="uwb_pose_publisher.py" args="$(arg world_frame) TODO" output="screen" required='true' if="$(eval arg('publish_tfs') == true)"/>
    </group>
</launch>